beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -20,36 +20,12 @@ type WeakSubjectivityResponse struct {
 	} `json:"data"`
 }
 
-type BlockResponseJson struct {
-	Data *SignedBeaconBlockJson `json:"data"`
-}
-
-type BlockV2ResponseJson struct {
-	Version             string                            `json:"version" enum:"true"`
-	Data                *SignedBeaconBlockContainerV2Json `json:"data"`
```

Comment:
You can clear these as well. From what I can see the following have no usages:
`SignedBeaconBlockContainerV2Json`
`SignedBlindedBeaconBlockContainerJson`
`SignedBlindedBeaconBlockContentsContainerJson`